api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -436,6 +436,14 @@ enum ProductType {
   PRODUCT_TYPE_EUB = 2;
 }
 
+// SupportType if the type of support offered.
+enum SupportType {
+  // SUPPORT_TYPE_FREE is the free tier support
+  SUPPORT_TYPE_FREE = 0;
```

Comment:
Could we avoid adding meaning to the zero enum?

* https://cloud.google.com/apis/design/design_patterns#enum_default_value
* https://protobuf.dev/programming-guides/style/#enums